app/controllers/development_controller.rb

```diff
@@ -335,6 +342,14 @@ def membership_request_from_user
     @membership_request_from_user
   end
 
+  def pending_invitation
+    pending_invitation = InvitationService.invite_to_group(recipient_emails: ['judd@example.com'],
+                                                           message: 'Come and join the group!',
+                                                           group: test_group,
+                                                           inviter: patrick)
+    pending_invitation
```

Comment:
Confused by this; we want to memoize it I think, so it doesn't get created more than once? Definitely don't need to store in a variable and then return that variable I don't think.

```
@pending_invitation ||= InvitationService.invite_to_group(...)
```